docs/contract-management.md

```diff
@@ -0,0 +1,71 @@
+
+#### Contracts
+Contracts in this context, are POs of a certain scale, originating in SAP and resolved from common library. 
+
+Common library will provide list available contracts for a specific project (project master), and provides the contract number, contract name and the company name.
+
+#### Affiliate account
+A guest account in the company Azure AD. This means the user can sign in to the company systems using their own e-mail. To get this account; **1.** the e-mail has to be invited; **2.** the user receives a mail with a link; **3** the user has to follow the link and agree to the company terms.
+
+# Request flow
+<p align="center">  <img src="https://github.com/equinor/fusion-app-resources/blob/docs/contract-management/docs/images/contracts-request-flow.svg">  </p>
+
+## The overall process
+
+<p align="center">  <img src="https://github.com/equinor/fusion-app-resources/blob/docs/contract-management/docs/images/contracts-general-process.svg">  </p>
+
+>  ### 1. Allocation
+> The contract is allocated by a company employ with correct permission. This permission is granted to persons with procurement positions in the selected project, or super-users with said permission for all projects.
+> Contracts are listed from common library for the selected project. 
+>
+> **If the contract does not appear, it has to be added to common library. This can be done by raising a Service Now ticket***
+
+> ### 2. Manage Personnel
+> ![#d66dd3](https://placehold.it/15/d66dd3/000000?text=+) CR/PR can define personnel that will be available for the request process. 
+> This step is isolated from the request process, to give a clear view of users that is involved in the contract, and a workspace to manage general user info, like relevant disciplines, davinci code, linked in profile etc.
+>
+> Once a user is added to the personnel list, the system will do a check to see if that user has an affiliate account. This is indicated with a **gray**/**yellow**/**green** icon. 
+> 
+> ![#a6a6a6](https://placehold.it/15/cacaca/000000?text=+) **Gray** indicates the user does not have an account.</br>
+> ![#f0b913](https://placehold.it/15/f0b913/000000?text=+) **Yellow** means the user has **not** accepted the invitation yet, so he cannot sign in.</br>
+> ![#1fcf00](https://placehold.it/15/1fcf00/000000?text=+) **Green** the user can sign in and use company services. 
+
+> ### 3.1 Requests - create/submitt
+> ![#d66dd3](https://placehold.it/15/d66dd3/000000?text=+) CR/PR and any other users with a position in the contract can create requests to add personnel to the contract.
+> If a non CR/PR user creates the request, the CR/PR has to approve for the request to be submitted to the ![#6f30a0](https://placehold.it/15/6f30a0/000000?text=+) CR for approval.
+>  
+>  **If a person is not available for requests, the user might not have a yellow/green status**
```

Comment:
Can a person be available for requests if the status is yellow? If so, a change has to be made in frontend

Reply:
just as well, but is technically possible. Might be subject to change later